graphql/schema/types/distro.graphql

```diff
@@ -291,6 +292,7 @@ type Distro {
   note: String!
   plannerSettings: PlannerSettings!
   provider: Provider!
+  providerAccount: String
```

Comment:
I have a question related to Kim's comment. If a user sets a provider account from the frontend but the following scenario is true

> actually can't be populated if using kernel-build since the default provider account is kernel-build

what happens? I also might just be misinterpreting what Kim is saying

Reply:
I spoke to kim there is a fixed list of approved provider accounts that exist in admin settings. `kernel-build` is not one of them and is just the default we use if we are not using one of the other provider account